charts/aws-ebs-csi-driver/templates/_node.tpl

```diff
@@ -53,6 +53,7 @@ spec:
         {{- with .Values.node.tolerations }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- include "aws-ebs-csi-driver.daemonset-tolerations" . | nindent 8 }}
         - key: "ebs.csi.aws.com/agent-not-ready"
           operator: "Exists"
```

Comment:
Can we move this `agent-not-ready` taint into the recommended tolerations as well, so it's applied to both linux and windows? Otherwise lgtm

Reply:
To confirm, it is a bug that it has not been applied to Windows this whole time right?

Reply:
Correct - that's the taint that is used by our [startup taint feature](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/install.md#configure-node-startup-taint), which isn't in any way linux-specific, so the lack of this toleration is just a straight up bug we missed when originally implementing that feature.